sigs/sig-marketing/README.md

```diff
@@ -35,3 +39,11 @@ Sig marketing meetings are held twice monthly. Meetings can be found in the [Arg
 | Jenny Blekh | Octopus Deploy | jenny.blekh@octopus.com |
 | Meghan Murphy | Intuit | Meghan_Murphy@intuit.com |
 | Wojtek Cichoń | Akuity | wojtek@akuity.io  |
+
+## Interested Party
```

Comment:
How are we planning to use this? This seems like a frequently changing list and requires a lot of maintenance.

Reply:
@terrytangyuan This is only for sig-marketing and basically acts as a way for people to document their interest in participating.